tests/attr/test_lrp.py

```diff
@@ -0,0 +1,314 @@
+            return self.relu(self.lin(input))[0].unsqueeze(0)
+
+    input = torch.tensor([[1.0, 2.0], [1.0, 3.0]])
+    model = MyModel(inplace)
+
+    return model, input
+
+
+class Test(BaseTest):
+    def test_lrp_creator(self):
+        model, _ = _get_basic_config()
+        model.conv1.rule = 1
+        self.assertRaises(TypeError, LRP, model)
+
+    def test_lrp_creator_activation(self):
+        model, inputs = _get_basic_config()
+        model.add_module("sigmoid", nn.Sigmoid())
+        lrp = LRP(model)
+        self.assertRaises(TypeError, lrp.attribute, inputs)
+
+    def test_lrp_basic_attributions(self):
+        model, inputs = _get_basic_config()
+        logits = model(inputs)
+        _, classIndex = torch.max(logits, 1)
+        lrp = LRP(model)
+        relevance, delta = lrp.attribute(
+            inputs, classIndex.item(), return_convergence_delta=True
+        )
+        self.assertEqual(delta.item(), 0)
+        self.assertEqual(relevance.shape, inputs.shape)
+
+    def test_lrp_simple_attributions(self):
+        model, inputs = _get_simple_model()
+        model.eval()
+        model.linear.rule = EpsilonRule()
+        model.linear2.rule = EpsilonRule()
+        lrp = LRP(model)
+        relevance = lrp.attribute(inputs)
+        assertTensorAlmostEqual(
+            self, relevance, torch.tensor([18 / 108, 36 / 108, 54 / 108])
+        )
+
+    def test_lrp_simple_attributions_batch(self):
+        model, inputs = _get_simple_model()
+        model.eval()
+        model.linear.rule = EpsilonRule()
+        model.linear2.rule = EpsilonRule()
+        lrp = LRP(model)
+        inputs = torch.stack((inputs, 3 * inputs))
+        relevance, delta = lrp.attribute(inputs, return_convergence_delta=True)
+        self.assertEqual(relevance.shape, inputs.shape)
+        self.assertEqual(delta.shape[0], inputs.shape[0])
+
+    def test_lrp_simple_repeat_attributions(self):
+        model, inputs = _get_simple_model()
+        model.eval()
+        model.linear.rule = GammaRule()
+        model.linear2.rule = Alpha1_Beta0_Rule()
+        output = model(inputs)
+        lrp = LRP(model)
+        _ = lrp.attribute(inputs)
+        output_after = model(inputs)
+        assertTensorAlmostEqual(self, output, output_after)
+
+    def test_lrp_simple_inplaceReLU(self):
+        model_default, inputs = _get_simple_model()
+        model_inplace, _ = _get_simple_model(inplace=True)
+        for model in [model_default, model_inplace]:
+            model.eval()
+            model.linear.rule = EpsilonRule()
+            model.linear2.rule = EpsilonRule()
+        lrp_default = LRP(model_default)
+        lrp_inplace = LRP(model_inplace)
+        relevance_default = lrp_default.attribute(inputs)
+        relevance_inplace = lrp_inplace.attribute(inputs)
+        assertTensorAlmostEqual(self, relevance_default, relevance_inplace)
+
+    def test_lrp_simple_tanh(self):
+        class Model(nn.Module):
+            def __init__(self):
```

Comment:
nit: perhaps you can add this here and we can reuse it ?
https://github.com/pytorch/captum/blob/master/tests/helpers/basic_models.py#L405